src/components/Chat/ChatMessage.tsx

```diff
@@ -323,8 +325,9 @@ export const ChatMessage: FC<Props> = memo(
       >
         <div
           className={classNames(
-            'relative m-auto flex h-full p-4 md:gap-6 md:py-6 lg:px-0',
-            { 'md:max-w-2xl xl:max-w-3xl': !isChatFullWidth },
+            'relative m-auto flex h-full md:gap-6 md:py-6 lg:px-0',
+            !isChatFullWidth && 'md:max-w-2xl xl:max-w-3xl',
+            isOnlySmallScreen() || isOverlay ? 'px-4 py-3' : 'p-4',
```

Comment:
Is it possible to use breakpoint classes instead of just plain css classes?

Reply:
We depend here also on overlay, not only on screen size.